assets/wix/Product.wxs

```diff
@@ -135,8 +133,11 @@
       <Publish Dialog="MyExitDialog" Control="Finish" Order="1" Event="DoAction" Value="LaunchApplication">LAUNCHAPPONEXIT=1</Publish>
     </UI>
     <!-- Prerequisites -->
-    <Condition Message="Supported only on Windows 7 and above">
-      <![CDATA[ Installed OR $(var.MinOSVersionSupported) ]]>
+    <Condition Message="Supported only on Windows 8.1 and above">
```

Comment:
Since 8.1 is out of support very soon, should this be `Windows 10 and newer`?

Reply:
Not from _within_ the MSI. See the conversation on #18854. The only way we could do that is start shipping the MSI wrapped in the burn bundle (which seems otherwise unused currently) and make the MSI difficult to install without using the burn bundle. Most products like Office, Visual Studio (when we used MSIs primarily, which we haven't since I architected the newer, faster, more-reliable installer), etc. do that. You can override (can't fully block it), which basically means, "I accept the license...and the consequences."